test/sbroker_SUITE.erl

```diff
@@ -204,7 +204,7 @@ skip_down_match(_) ->
             Self = self(),
             {await, Ref, _} = sbroker:async_ask_r(Broker, Self, {Self, Ref}),
             sys:resume(Broker),
-            {drop, _} = sbroker:await(Ref, ?TIMEOUT),
+            _ = sbroker:await(Ref, ?TIMEOUT),
```

Comment:
^ Why the change to the matched return value?

Reply:
Tests fail. Should we look closer into it? Or just assume it's working as is and move on?

Reply:
I think we should look into it; I believe they were working before (e.g. under OTP 19.)

Reply:
Alas, it appears I've at one point known about this issue:

https://github.com/fishcakez/sbroker/pull/5#issuecomment-500453110

Reply:
So, is this just a "test fail, but code doesn't" type of situation, or is there something we REALLY want to fix, here?

Reply:
I'll try to find time to look at it later, then. I don't want to sweep it under the rug "just because".

Reply:
As discussed, we'll go with "skip this test case": dc47fe8